pkg/env/env.go

```diff
@@ -441,13 +445,22 @@ func (e *testEnv) execFeature(ctx context.Context, t *testing.T, featName string
 				}
 				ctx = assess.Func()(ctx, t, e.cfg)
 			})
+
+			if e.cfg.FailFast() && t.Failed() {
```

Comment:
Since `t.Failed()` is checked, is this feature dependent on test writer calling `t.FailNow()` (or similar)? If so, it should be stated in the README.md that the test writer must signal failure.

Reply:
Yes. Right now the decision of if things are to fail or not is upto the test writer. With the flag enabled, they can terminate the workflow where they see fit. This way, we need not infer and make decision for them and the writers can do it themselves. I will update the document here as well + the Readme too.